src/popup/pages/SignMessage.vue

```diff
@@ -1,78 +1,41 @@
-<template>
-  <div class="sign-message">
-    <div class="section-title">
-      Sign message for
-    </div>
-
-    <div class="url-bar">
-      {{ callbackOrigin }}
-    </div>
-
-    <div class="section-title">
-      Message
-    </div>
-
-    <div class="tip-note-preview">
-      {{ $route.query.message }}
-    </div>
-
-    <BtnMain @click="sendAddress">
-      {{ $t('pages.tipPage.confirm') }}
-    </BtnMain>
-    <BtnMain @click="openCallbackOrGoHome(false)">
-      {{ $t('pages.tipPage.cancel') }}
-    </BtnMain>
-  </div>
-</template>
-
 <script lang="ts">
-import { defineComponent } from '@vue/composition-api';
+import { defineComponent, onMounted } from '@vue/composition-api';
 import { useDeepLinkApi, useSdk } from '../../composables';
-import BtnMain from '../components/buttons/BtnMain.vue';
+import { MODAL_MESSAGE_SIGN, handleUnknownError } from '../utils';
 
 export default defineComponent({
   name: 'SignMessage',
-  components: {
-    BtnMain,
-  },
   setup(props, { root }) {
-    const { openCallbackOrGoHome } = useDeepLinkApi({ router: root.$router });
-    const { getSdk } = useSdk({ store: root.$store });
-
-    const sendAddress = async () => {
-      const sdk = await getSdk();
-      const signature = await sdk.signMessage(root.$route.query.message);
-      const signatureHex = Buffer.from(signature).toString('hex');
-      openCallbackOrGoHome(true, { signature: signatureHex });
-    };
-
-    return {
-      sendAddress,
-    };
+    onMounted(async () => {
+      const { callbackOrigin, openCallbackOrGoHome } = useDeepLinkApi({ router: root.$router });
+      const { getSdk } = useSdk({ store: root.$store });
```

Comment:
The instantiation of the composables should be put at the beginning of the `setup` to follow common way we do it within other components.